test/cpp/end2end/end2end_test.cc

```diff
@@ -1131,24 +1131,34 @@ TEST_P(End2endTest, CancelRpcBeforeStart) {
   }
 }
 
-// TODO(https://github.com/grpc/grpc/issues/21263): stop using timed sleeps to
-// synchronize cancellation semantics.
 TEST_P(End2endTest, CancelDelayedRpc) {
```

Comment:
Nit: For precision and to be consistent with the naming of the test directly above, we could name this CancelRpcAfterStart

Reply:
Done.